generate_abstract-bazel.mk

```diff
@@ -5,5 +5,5 @@ bazel-generate_abstract:
 
 .PHONY: bazel-generate_abstract_mock_area
 bazel-generate_abstract_mock_area: bazel-generate_abstract
-	cp $(RESULTS_DIR)/../mock_area/$(DESIGN_NAME).lef $(RESULTS_DIR)/
```

Comment:
hmmm.... was the removal of mock_area intentional here?

Reply:
If I understand this correctly, this target was meant to copy files from `mock_area` flow variant output directory to `base` (or other flow variant provided in one of the `build_openroad` attributes) output directory.

After my modifications `RESULTS_DIR` in the context of `mock_area` stages already points to the directory relevant to `mock_area` flow variant. We then have to build the path for the other flow variant from scratch.